pallets/xyk/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = 'pallet-xyk'
-version = '3.6.5'
+version = '3.7.0'
```

Comment:
i think this should be a major version change . 

Because it introduces a change which is non backward compatible.wdyt ?

Reply:
Not sure if a pallet Config belongs to public API. But we can bump the major version.

Reply:
sure it does. you can upgrade pallet-xyk version in runtime as the config changes.